### PR TITLE
Drop duplicate CompilerTest

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: test/files/run/compiler-asSeenFrom.scala [hunk 1]
```diff
@@ -1,56 +1,10 @@
 /*
  * filter: inliner warning; re-run with
```
Comment (som-snytt) [inline test/files/run/compiler-asSeenFrom.scala]: The unmodified test succeeds without this filter.

File: test/files/run/t7096.scala [hunk 2]
```diff
@@ -3,53 +3,6 @@
  */
```
Comment (som-snytt) [inline test/files/run/t7096.scala]: Ditto.

File: test/files/run/compiler-asSeenFrom.scala (hunk 1 above)
Comment (dwijnand) [reply test/files/run/compiler-asSeenFrom.scala]: Done.

Time for a partest front matter linter?  😄

**Review by som-snytt — APPROVED** [review]
We need a mechanism to trigger obsolescence of workarounds, like `failAfter(someVersion)`.

**Comment by dwijnand** [comment]
Maybe we should do a thing like Seth does in the community build, and leave a comment describing the cause for the workaround, and maybe even the date.

**Pull request merged**